solr/solr-ref-guide/src/requesthandlers-and-searchcomponents-in-solrconfig.adoc

```diff
@@ -18,34 +18,84 @@
+
+[source,xml]
+----
+<requestHandler name="/spell" class="solr.SearchHandler" startup="lazy">
+ ...
+</requestHandler>
+----
+
+== Configuring request handlers
+There are 3 ways to configure request handlers inside their definitions and another 3 ways to configure them somewhere else.
 
-If you have another request handler defined, you would send your request with that name. For example, `/update` is a request handler that handles index updates (i.e., sending new documents to the index). By default, `/select` is a request handler that handles query requests.
+=== Request parameters (GET and POST)
+The easiest and most flexible way is to provide parameters with standard GET or POST requests.
 
-Request handlers can also process requests for nested paths of their names, for example, a request using `/myhandler/extrapath` may be processed by a request handler registered with the name `/myhandler`. If a request handler is explicitly defined by the name `/myhandler/extrapath`, that would take precedence over the nested path. This assumes you are using the request handler classes included with Solr; if you create your own request handler, you should make sure it includes the ability to handle nested paths if you want to use them with your custom request handler.
+Here is an example of sending parameters `id`, `fl`, and `wt` to `/select` Search Handler.
+Notice the URL-encoded space (as +) for the values of `fl` parameter.
 
-It is also possible to configure defaults for request handlers with a section called `initParams`. These defaults can be used when you want to have common properties that will be used by each separate handler. For example, if you intend to create several request handlers that will all request the same list of fields in the response, you can configure an `initParams` section with your list of fields. For more information about `initParams`, see the section <<initparams-in-solrconfig.adoc#initparams-in-solrconfig,InitParams in SolrConfig>>.
+[source,text]
+----
+http://localhost:8983/solr/techproducts/select?q=id:SP2514N&fl=id+name&wt=xml
+----
 
-=== SearchHandlers
+And here is an example of parameters being sent through the POST form to `/query` Search Handler using <<json-request-api.adoc#json-request-api,JSON Request API>>.
+
+[source,bash]
+----
+curl http://localhost:8983/solr/techproducts/query -d '
+{
+  "query" : "memory",
+  "filter" : "inStock:true"
+}'
+----
 
-The primary request handler defined with Solr by default is the "SearchHandler", which handles search queries. The request handler is defined, and then a list of defaults for the handler are defined with a `defaults` list.
+Either way, the parameters are extracted and combined with other options explained below.
 
-For example, in the default `solrconfig.xml`, the first request handler defined looks like this:
+=== defaults, appends, and invariants
```

Comment:
I think these should be capitalized. In the sub-sections here you did not capitalize "defaults" but you did "Appends" - it should be consistent, and even though those terms are the XML names we can also refer to the concepts of default and append, etc, as words and capitalize them in headings.